packages/core/src/lib/history.ts

```diff
@@ -0,0 +1,110 @@
+import { LogResult, simpleGit } from 'simple-git';
+import {
+  CoreConfig,
+  Format,
+  PersistConfig,
+  UploadConfig,
+  uploadConfigSchema,
+} from '@code-pushup/models';
+import { getCurrentBranchOrTag, safeCheckout } from '@code-pushup/utils';
+import { collectAndPersistReports } from './collect-and-persist';
+import { GlobalOptions } from './types';
+import { upload as uploadCommandLogic } from './upload';
```

Comment:
Why rename it?

Reply:
It's a leftover as I had the upload property of the core config destructed. Thx for catching it!